src/script/script.go

```diff
@@ -32,6 +32,7 @@ func ActivateDryRun() {
 
 // OpenBrowser opens the default browser with the given URL.
 func OpenBrowser(url string) {
+  fmt.Printf("Opening URL: %s\n", url)
```

Comment:
Sorry for the not-helpful-at-all error message from `goimports`, @tallpants! Goimports gets hung up that you use two spaces for indentation here, and insists that you use a single tab character instead. You can run `gofmt -s -w src` to fix this in your branch. You should have `gofmt` since it is a part of the standard Go toolchain. This suggested fix here on GitHub should take care of this issue as well:  
```suggestion
	fmt.Printf("Opening URL: %s\n", url)
```

Reply:
Btw as a reference, here is the documentation for `goimports` including instructions for installing it, in case you need it later again: https://godoc.org/golang.org/x/tools/cmd/goimports